File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/While_loops/Factorial.java
//Printing factorial of the entered number 

import java.util.*;

public class Factorial
{
    public static void main(String args[])
    {
        Scanner in=new Scanner(System.in);
        int N,f=1;
        System.out.println("Enter the value of N");
        N=in.nextInt();
        int i=1;
        while(i<=N)
        {
            f=f*i;
            i++;
        }

        System.out.println("The factorial of " + N + " is " + f);
    }
}

File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/While_loops/factorial.cpp
//Printing factorial of the entered number 

#include<iostream>
using namespace std;

int main()
{
        int N,f=1;
        cout<<"Enter the value of N"<<endl;
        cin>>N;
        int i=1;
        while(i<=N)
        {
            f=f*i;
            i++;
        }

        cout<<"The factorial of " << N << " is " << f;
}
